### ISSUE TITLE
Does not detect typescript is installed and will not install

### ISSUE THREAD

**Issue by smlombardi** [palantir/tslint#1002]
On my Mac I have typescript 1.8.2 installed.  I verified it with 

```
charizard:~ stevel$ tsc --version
Version 1.8.2
```

however, if I try to install tslint, I get

```
npm WARN EPEERINVALID tslint@3.5.0 requires a peer of typescript@>=1.7.3 but none was installed.
```
**UPDATE**
I also tried installing tslint@next, but that also fails:
```
npm WARN EPEERINVALID tslint@3.5.0-dev.1 requires a peer of typescript@>=1.7.3 || >=1.8.0-dev || >=1.9.0-dev but none was installed.
```

**Comment by JKillian** [comment]
Hi @smlombardi, please see #815 which is the same issue as this. We haven't ben able to figure out yet why NPM gives that warning, however things should still work correctly as long as they're both installed despite the ominous warning.
